docs/travis.md

```diff
@@ -0,0 +1,102 @@
+# Travis CI
+
+The Zulip server uses [Travis CI](https://travis-ci.org/) for its
+continuous integration.  This page documents useful tools and tips to
+know about when using Travis CI and debugging issues with it.
+
+## Goals
+
+The overall goal of our Travis CI setup is to catch as many possible
+future bugs in Zulip as possible, minimizing delay and false positives
+that waste the time of developers around the Zulip community
+debugging.  There are a few implications of this overall goal:
```

Comment:
I want something like a preposition to connect "debugging".

"that waste the time of developers around the Zulip community through debugging"?
"... with debugging"?